src/Logging/Log.php

```diff
@@ -36,8 +36,8 @@ class Log
      */
     public static function get_logger()
     {
-        Deprecation::notice('5.0', 'Use Injector::inst()->get(\'Logger\') instead');
-        return Injector::inst()->get('Logger');
+        Deprecation::notice('5.0', 'Use Injector::inst()->get(LoggerInterface::class) instead');
```

Comment:
I feel like deprecating APIs for 5 before 4 is even released is a bit silly - we should trash them now.